core/wiki/macros/translink.tid

```diff
@@ -3,13 +3,13 @@ tags: $:/tags/Macro
 
 \define translink(title,mode:"block")
 \whitespace trim
-<div style="border:1px solid #ccc; padding: 0.5em; background: black; foreground; white;">
+<div class="tc-pre-background">
```

Comment:
Rather than reusing generic classes, I wonder if we might be better off introducing a new class for the translink macro "tc-translink", and then targetting the child DIV and the h1 with CSS selectors. The motivation is that it would make it easier for end users to restyle the macro output without having to alter the macro itself.